src/commands/moderation/ban.ts

```diff
@@ -63,7 +63,7 @@ export const command: Command = {
 
         const member: GuildMember = <GuildMember> ctx.options.getMember("user") ?? null;
         const user: User = ctx.options.getUser("user"); // Never null
-        const reason: string = ctx.options.getString("reason") ?? null;
+        const reason: string = `${ctx.user}: ${ctx.options.getString("reason") ?? "No reason"}`;
```

Comment:
This is added to all moderation commands to make them more transparent, showing the mod that used the commands.